pkg/controller/modeler/modeler_test.go

```diff
@@ -219,4 +219,43 @@ func TestRebuildModel(t *testing.T) {
 			t.Errorf("expected %v to be %v", got, want)
 		}
 	}
+
+	// Ensure we hit the ceiling
+	{
+		if err := db.RawDB().Exec("TRUNCATE realm_stats").Error; err != nil {
+			t.Fatal(err)
+		}
+
+		line := make([]uint, 24)
+		for i := range line {
+			line[i] = uint(10000 * i)
```

Comment:
some function on hardMax? Otherwise you'll have to change this test when you change the constant.